tasks/check-data-keys/check-data-keys.yaml

```diff
@@ -24,52 +29,46 @@ spec:
       description: The workspace where the data JSON file resides
   steps:
     - name: check-data-keys
-      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+      image: quay.io/konflux-ci/release-service-utils@sha256:b8a4a4ff239d7789029a7bbd8c89d41453107d4391c1aaf7ea158f343d5624f6
       args: ["$(params.systems[*])"]
       script: |
         #!/usr/bin/env bash
         set -ex
 
-        KEYS_JSON='{
-            "releaseNotes": [
-                "product_id",
-                "product_name",
-                "product_version",
-                "product_stream",
-                "cpe",
-                "type",
-                "content.images",
-                "synopsis",
-                "topic",
-                "description",
-                "solution",
-                "references"
-            ],
-            "cdn": [
-                "env"
-            ]
-        }'
-
         if [ ! -f "$(workspaces.data.path)/$(params.dataPath)" ] ; then
             echo "No data JSON was provided."
             exit 1
         fi
 
-        RC=0
+        if ! curl -s --fail-with-body "$(params.schema)" -o /tmp/output ; then
```

Comment:
Maybe put it in /tmp/schema or something more intuitive?

Reply:
fixed